### ISSUE TITLE
aarch64 support? " cannot execute binary file: Exec format error:"

### ISSUE THREAD

**Issue by TheRealDirtyMike** [tidusjar/Ombi#1830]
#### Ombi build Version:

V 3.0.2534

#### Update Branch:

Open Beta

#### Media Sever:

N/A

#### Media Server Version:

N/A

#### Operating System:
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.2 LTS
Release:	16.04
Codename:	xenial

Linux DoucheCanoe 3.14.79-108 #1 SMP PREEMPT Mon Feb 27 23:18:26 BRT 2017 aarch64 aarch64 aarch64 GNU/Linux

#### Ombi Applicable Logs (from `/logs/` directory or the Admin page):

None available

#### Problem Description:
Unable to run Ombi as I am getting the below error message. The file is executable and I am running on a 64bit system. (Odroid C2) Armv8 64bit chipset. I have run through the same install steps on a different Ubuntu machine using a traditional 64bit chipset that is not an ARM and it is working correctly.

**Non Working ARMv8 64bit processor:**
root@DoucheCanoe:/opt/Ombi# ./Ombi
-su: ./Ombi: cannot execute binary file: Exec format error 

root@DoucheCanoe:/opt/Ombi# uname -a
Linux DoucheCanoe 3.14.79-108 #1 SMP PREEMPT Mon Feb 27 23:18:26 BRT 2017 aarch64 aarch64 aarch64 GNU/Linux

root@DoucheCanoe:/opt/Ombi# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.2 LTS
Release:	16.04
Codename:	xenial

**Working x86_64 processor:**
root@IL1GOCHPYMIKE01:/opt/Ombi# uname -a       
Linux IL1GOCHPYMIKE01 4.4.0-34-generic #53-Ubuntu SMP Wed Jul 27 16:06:39 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

root@IL1GOCHPYMIKE01:/opt/Ombi# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.1 LTS
Release:	16.04
Codename:	xenial
root@IL1GOCHPYMIKE01:/opt/Ombi# 

#### Reproduction Steps:

running ./Ombi

**Comment by TheRealDirtyMike** [comment]
I assume that even 64bit ARM builds are not supported yet as I am seeing a few other 'issues' referencing using ARM. I guess this issue can be closed out. Hopefully ARM support is added soon as it seems there are a number of people trying to use PI's and other development boards that use arm.

**Comment by goldenpipes** [comment]
does this help at all? https://github.com/tidusjar/Ombi/pull/1808? i think the issue is something win the dotnetcore2.0 for arm.

**Comment by TheRealDirtyMike** [comment]
Oh nice. Yea I will have to check out that one later when I can. Thanks. Will update you.

**Comment by FeatherKing** [comment]
Did you build your own version of dotnet core? The common one I see floating around in documentation is 32 bit, where ombi seems to require 64bit.

Based on the dotnet github page, it should be possible to compile a dotnet core client targeting for arm64: https://github.com/dotnet/coreclr/tree/master/cross

**Comment by tidusjar** [comment]
It is possible, I need to make some changes before I do this.

**Comment by tidusjar** [comment]
Check the latest builds

**Comment by TheRealDirtyMike** [comment]
Forgive me if I am missing something here. I attempted to install just now using the build package "linux-arm.tar.gz" from the AppVeyor page and it is still not working. Different error now though. However the error doesn't make sense.. I unzip the tar ball and make the Ombi file executable, however when I attempt to run it, I am told that the file or directory does not exist... Super weird. Here is what I am seeing.
////////////////
root@DoucheCanoe:/opt/Ombi# ls -la | grep Ombi
-rwx--x--x  1 root root    67684 Jan 18 04:15 Ombi
output omitted for brevity

root@DoucheCanoe:/opt/Ombi# ./Ombi
-su: ./Ombi: No such file or directory
root@DoucheCanoe:/opt/Ombi# 

Install steps followed as usual. Am I pulling the wrong package or location?

**Comment by tidusjar** [comment]
Make sure the working directory is set to where you unzipped the files.
Also make sure ombi has the executable permission.

**Comment by TheRealDirtyMike** [comment]
All confirmed. I have done nothing different with this installation attempt from when I installed on a working Linux VM. I have unzipped the package in the /opt/Ombi directory and make the Ombi file executable and am attempting to run the file from the same directory. I am downloading the ARM package when I get this error. I do not get this error when using the standard Linux package, however that one does not work as expected due to not being ARM compatible.

root@DoucheCanoe:/opt/Ombi# pwd
/opt/Ombi
root@DoucheCanoe:/opt/Ombi# ls -la | grep Ombi
-rwx--x--x  1 root root    67684 Jan 18 04:15 Ombi
root@DoucheCanoe:/opt/Ombi# 
root@DoucheCanoe:/opt/Ombi# ./Ombi
-su: ./Ombi: No such file or directory
